src/Altinn.Platform/Altinn.Platform.Events/Tests/TestingControllers/SubscriptionControllerTests.cs

```diff
@@ -225,6 +225,34 @@ public async void Post_GivenSubscriptionOrganizationWithValidSubject_ReturnsCrea
                 Assert.Equal(HttpStatusCode.Created, response.StatusCode);
             }
 
+            /// <summary>
+            /// Post invalid subscription for organization
+            /// Expected result:
+            /// Returns HttpStatus badrequest
+            /// Success criteria:
+            /// The response has correct status and correct responseId.
+            /// </summary>
+            [Fact]
+            public async void Post_GivenSubscriptionOrganizationWithInValidSubject_ReturnsBadRequest()
```

Comment:
It looks like this new test fails to test the new code. SonarCloud is showing low coverage for new code.

Reply:
updated test